logging/logging.go

```diff
@@ -252,6 +252,69 @@ var detectedResource struct {
 	once sync.Once
 }
 
+func detectGAEResource() *mrpb.MonitoredResource {
```

Comment:
For breaking changes we will want to make these in a v2 directory. This ensures compatibility for GOPATH users as well as modules.